### ISSUE TITLE
Enforce `.items()` usage in loops

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#997]
# Rule request

## Thesis

When we see a code like this:

```python
for code in country_map:
      name = country_map[code]
      # do something with name
```

We should raise a refactoring violation. Why?
Because accessing `country_map[...]` in a loop indicates that it is pythonic.

That's how it should be done instead:

```python
for code, name in country_map.items():
    # do something with name
    pass
```

Origin: https://deepsource.io/blog/8-new-python-antipatterns
